testdata/ansible/memcached-operator/watches.yaml

```diff
@@ -3,5 +3,5 @@
 - version: v1alpha1
   group: cache.example.com
   kind: Memcached
-  role: memcached
+  playbook: playbooks/memcached.yml
```

Comment:
@asmacdo @fabianvf, 

I change the sample to create the api with `--generate-role` and  `--generate-playbook`. Should not create both instead of the latest one only?

Reply:
If you create both, the playbook is the only thing that needs to be in the watches file. The role is still used though, it is executed by the playbook.